actions/class.TestRunner.php

```diff
@@ -291,21 +295,26 @@ protected function afterAction($withContext = true) {
      */
 	public function index() {
 	    $this->beforeAction();
+        $config = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiTest')->getConfig('testRunner');
 	    $session = $this->getTestSession();
-	    
+
+        if (isset($config['reset-timer-after-resume']) && $config['reset-timer-after-resume']) {
+            $sessionStateService = $this->getServiceManager()->get('taoQtiTest/SessionStateService');
+            $sessionStateService->updateTimeReference($session);
```

Comment:
Why are you re-updating the time ? can't you just do the resume here and not in the beforeAction ?